webhook_server/libs/handlers/pull_request_handler.py

```
@@ -981,10 +1020,17 @@ async def _delete_registry_tag_via_regctl(
-                f"Failed to delete tag: {repository_full_tag}. Please delete it manually.",
-            )
+            try:
+                await github_api_call(
+                    pull_request.create_issue_comment,
+                    f"Failed to delete tag: {repository_full_tag}. Please delete it manually.",
+                    logger=self.logger,
+                    log_prefix=self.log_prefix,
+                )
+            except Exception:
+                self.logger.exception(
+                    f"{self.log_prefix} Tag cleanup succeeded, but PR notification failed: {repository_full_tag}"
```

Ruff (0.15.9)

[warning] Logging statement uses f-string

(G004)